### DISCUSSION TITLE
Option to always collapse certain columns

### DISCUSSION THREAD

**Discussion by eswachsman** [rappasoft/laravel-livewire-tables#964] [Feature Requests]
It would be awesome if there would be an option to always collapse certain columns, maybe `collapse()` , just like you have `collapseOnTablet()` and `collapseOnMobile()`. 

I have certain very long fields, and even on desktop it would make sense to have them collapsed.

**Comment by dave2084** [comment]
I'd like to see this, even better if would could pass a bootstrap/tailwind breakpoint value to hide on.

**Comment by lrljoe** [comment]
This is being worked on [Here](https://github.com/rappasoft/laravel-livewire-tables/pull/1047) no idea on timescale, as it is another contributor working on it
